guide/english/java/generics/index.md

```diff
@@ -135,4 +134,27 @@ public class GenericsErasure
 
 When you compile some code against a generic type or method, the compiler works out what you really mean (i.e. what the type argument for T is) 
 and verifies at compile time that you're doing the right thing, but the emitted code again just talks in terms of java.lang.Object - the compiler generates extra casts where necessary. 
-At execution time, a List<String> and a List<Date> are exactly the same the extra type information has been erased by the compiler.
\ No newline at end of file
+At execution time, a List<String> and a List<Date> are exactly the same the extra type information has been erased by the compiler.
```

Comment:
```suggestion
At execution time, a List(String) and a List(Date) are exactly the same the extra type information has been erased by the compiler.
```
Should be a better alternative, the previous way it was done was making it be processed like HTML tags. If you want to keep the angled brackets, use the html codes for it

Reply:
The angled brackets are a part of the Java syntax, so cannot be changed. I have changed the brackets to use HTML codes. This is my first time contributing to a project on github, so I hope that I have done things correctly. Any help or feedback is appreciated.

Reply:
Since this is just a markdown file, you can do:
```
At execution time, a `List<String>` and a `List<Date>` are exactly the same the extra type information has been erased by the compiler.
```

Reply:
I have made the changes.